src/slices/donation/types.ts

```diff
@@ -12,6 +12,7 @@ export type DonationRecipient = {
 };
 
 type BaseDonationDetais = {
+  isInWidget: boolean;
```

Comment:
no need for extra `isInWidget` property as this can be determined by `source`
```js
< isInWidget={details.source === "bg-widget"} />
```

Reply:
I always understood the `bg-` part to mean "background", I just realized that it actually stands for "Better Giving".
Will fix

Reply:
Left the `isInWidget` logic in *StripePaymentStatus.tsx*, as there's no way to access `source` in that component (it is used as `return_url` param for Stripe, so *it* navigates us to this page (impossible to set props)).